traitsui/table_filter.py

```diff
@@ -412,7 +412,10 @@ class GenericTableFilterRuleEnabledColumn(ObjectColumn):
     def get_value(self, object):
         """ Returns the traits editor of the column for a specified object.
         """
-        return ['', '==>'][object.enabled]
+        if hasattr(object, 'enabled') and object.enabled:
+            return u'\u2714'
```

Comment:
What's that?

Reply:
Maybe it would be better spelled as `u'\N{HEAVY CHECK MARK}'`?

Reply:
Yep - will change.